### PR TITLE
Fix issues with day & night modes in python docs

### PR DESCRIPTION
**Pull request by isVoid**
Porting the fixes from rapidsai/cudf#11400

### CODE REVIEW

File: cpp/doxygen/Doxyfile [hunk 1]
```diff
@@ -1162,7 +1162,7 @@ HTML_STYLESHEET        =
 # list). For an example see the documentation.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_EXTRA_STYLESHEET  = rapids.css
+HTML_EXTRA_STYLESHEET  =
```
Comment (galipremsagar) [inline cpp/doxygen/Doxyfile:1165]: You will also need to set footer:
```
HTML_FOOTER = footer.html
```

**Review by galipremsagar — CHANGES_REQUESTED** [review]

File: cpp/doxygen/Doxyfile (hunk 1 above)
Comment (isVoid) [reply cpp/doxygen/Doxyfile:1165]: In cuspatial this property doesn't seem was set

Comment (galipremsagar) [reply cpp/doxygen/Doxyfile:1165]: I see, we probably will have to add that entry to let doxygen know to add the file to footer of all pages.

Comment (isVoid) [reply cpp/doxygen/Doxyfile:1165]: I was looking somewhere else. Fixed.

**Review by galipremsagar — APPROVED** [review]

**Review by harrism — APPROVED** [review]

**Comment by isVoid** [comment]
Working on a separate issue that will fix build issue with docs. May also include in this PR.

**Comment by isVoid** [comment]
@gpucibot merge

**Pull request merged**